components/admin/actions.tsx

```diff
@@ -182,11 +183,14 @@ export default function Actions({ data }: PageProps): JSX.Element {
     reason: Yup.string()
       .label('Reason')
       .oneOf(reasonOptions.map(({ value }) => value)),
-    applicationDate: Yup.string(),
-    informationReceived: Yup.string(),
+    applicationDate: Yup.string().notOneOf([INVALID_DATE], 'Invalid date'),
+    informationReceived: Yup.string().notOneOf([INVALID_DATE], 'Invalid date'),
```

Comment:
I think we need to centralise the logic with our date fields and the validation. Also, what's `Invalid date` if it's not `INVALID_DATE`?

Reply:
I took this from `address-history.tsx`, could definitely do with tidying up validation in general.